### PR TITLE
Use .word-break mixin along with max-width to enable word break within toast notifications.

### PR DESCRIPTION
**Pull request by sg00dwin**
Fixes https://github.com/openshift/origin-web-console/issues/1770

screen shot 2017-06-27 at 11 46 05 am
screen shot 2017-06-27 at 11 35 28 am

screen shot 2017-06-27 at 11 33 11 am

### CODE REVIEW

**Comment by sg00dwin** [comment]
@rhamilto, @jwforres

File: src/styles/_notifications.less [hunk 1]
```diff
@@ -8,7 +8,11 @@
 }
 
 .toast-notifications-list-pf {
+  @media(max-width: @screen-xxs-max) {
+    max-width: calc(100% - - 40px); // Set max-width when < 769, so that word-break functions on Firefox & IE.
```
Comment (rhamilto) [inline src/styles/_notifications.less]: Double minus?  @screen-xxs-max = 479, not 769.  Add additional comment on why 40px?

Comment (sg00dwin) [inline src/styles/_notifications.less]: the 40px accounts for the 20px left & right margin from `.container-fluid` 

Found that these are equal, but I'll switch to the other
calc(100% - - 40px) === calc(~"100% - 40px")

meant to use var @screen-xs-max, will update.

**Comment by sg00dwin** [comment]
@rhamilto updated pr

**Pull request merged**